source/local/local.go

```diff
@@ -157,7 +162,21 @@ func parseFeatures(lines [][]byte) map[string]string {
 			if len(lineSplit) == 1 {
 				features[key] = "true"
 			} else {
-				features[key] = lineSplit[1]
+				value := lineSplit[1]
```

Comment:
What I meant in https://github.com/kubernetes-sigs/node-feature-discovery/pull/1285#discussion_r1280770365 is that we should improve this parsing.

First, filter out all `# comments`, i.e. lines starting with a `#`). Plus probably do `stings.TrimSpace()` for each line to remove leading and trailing whitespace.

Second, for each "`#`" line do separate (generic) parsing to catch any `+<a-tag>=<a-value>`. This will make it easy to add new directives/tags in the future. Plus, we can catch unknown tags and log an error (e.g. if there is a typo like `+expiru-time=...`

Makes sense?

Reply:
#1293 and #1294 provide the first work item from above

Reply:
Thanks for the clarification.